### PR TITLE
XIVY-13821-webide-vsx

### PR DESCRIPTION
**Pull request by ivy-rew**
very early draft: 
feel free to review on my machine -> http://192.168.3.99/news/11.3#vsx-designer

### CODE REVIEW

**Review by ivy-cst — APPROVED** [review]
Nice :+1:

File: src/app/pages/news/11.3/08-vsx-designer.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Visual Studio Code Designer Extension (Alpha) 🧪 {#vsx-designer}
+
+We're doing first valiant explorative steps towards a web-based IDE.
```
Comment (weissreto) [inline src/app/pages/news/11.3/08-vsx-designer.md:3]: doing first -> doing the first

File: src/app/pages/news/11.3/08-vsx-designer.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- **Visual Studio Code**: We publish a first simple citizen developer toolset for Visual Studio Code as extension. Though we haven't decided yet about the final runtime environment, Visual Studio Code seems like a very promissing base to serve the Designer of the future.
```
Comment (weissreto) [inline src/app/pages/news/11.3/08-vsx-designer.md:5]: a first -> the first

Comment (weissreto) [reply src/app/pages/news/11.3/08-vsx-designer.md:5]: as extension -> as an extension

Comment (weissreto) [reply src/app/pages/news/11.3/08-vsx-designer.md:5]: promissing -> promising

File: src/app/pages/news/11.3/08-vsx-designer.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- **Editors**: Important editors, for Variables, Forms and Processes were developed to run natively in the Browser. Therefore, we already integrated these editors in our Designer extensions. This enables valiant early bird adopters to use the same featureset as within the classic Eclipse Designer.
```
Comment (weissreto) [inline src/app/pages/news/11.3/08-vsx-designer.md:6]: and Processes -> , and Processes

Comment (weissreto) [reply src/app/pages/news/11.3/08-vsx-designer.md:6]: in our -> into our

Comment (weissreto) [reply src/app/pages/news/11.3/08-vsx-designer.md:6]: featureset -> feature set / features

File: src/app/pages/news/11.3/08-vsx-designer.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- **Views**: With our custom Projects-View and the integrated Browsers, we already provide a welcoming environment to build first projects and simulate runs of processes.
+- **Workspaces**: Exploring dev-containers, we already gained trust, that this is an easy and powerful stack to share complex workspace setups. Focusing developers to work instantly, without the need to run manual setups.
+- **Cloud-ready**: With Github Codespaces, the Designer runs natively in the cloud. This gives us many options, to build process anywhere in no time.
```
Comment (weissreto) [inline src/app/pages/news/11.3/08-vsx-designer.md:9]: to build process -> to build processes

**Review by weissreto — APPROVED** [review]
Happy Grammarly, happy life ...

**Comment by ivy-rew** [comment]
thanks for the review :+1:

**Pull request merged**